client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -86,13 +92,26 @@ export const useCommandsArrayWpcom = ( {
 		};
 
 	const commandNavigation = useCommandNavigation();
-
 	const dispatch = useDispatch();
+
+	const { setEdgeCache } = useSetEdgeCacheMutation( {
+		onSettled: ( ...args ) => {
```

Comment:
I know it was like this already, but I think `onSuccess` is callback we want to be using. I used the dev tools to block requests to `public-api.wordpress.com/wpcom/v2/sites/*/hosting/edge-cache/active` and when you click the toggle control it will show a green success message.

Reply:
880aa3acd8f530d5b6dc3c04762192cafd6701d0